ui/StatusQ/src/StatusQ/Components/StatusDateGroupLabel.qml

```diff
@@ -10,6 +9,10 @@ StatusBaseText {
     property double previousMessageTimestamp
     property double messageTimestamp
 
+    readonly property int msInADay: 86400000
+    readonly property int lastMessageInDays:  Math.floor(previousMessageTimestamp / msInADay)
```

Comment:
this property is typed, so using `Math.floor` is not needed here at all

Reply:
Done